src/pages/index.js

```diff
@@ -45,7 +45,7 @@ import PagerDutyLogo from '../../content/assets/logos/pagerduty/pagerduty.inline
 import ArgoLogo from '../../content/assets/logos/argo-cd/argo.inline.svg';
 import JiraSoftwareLogo from '../../content/assets/logos/jira/jira.inline.svg';
 
-const SEO_TITLE = 'Batteries included Spotify Backstage';
+const SEO_TITLE = 'Internal Developer Portal built on Backstage by Spotify';
```

Comment:
I think im learning that this is what appears in google list.

So we should think about this one too.

Reply:
"Out-of-the-box Internal Developer Portal powered by open-source"